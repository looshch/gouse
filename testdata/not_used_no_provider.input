package p

// Any non-existent third party dependency works.
import "github.com/gorilla/mux"

// Tests if an import of an unused dependency breaks gouse.
func main() {
	notUsed0 := ""
}
