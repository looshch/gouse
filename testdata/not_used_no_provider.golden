package p

import "github.com/gorilla/mux" // Any non-existent dependency works.

func main() {
	var (
		notUsed0 = false; _ = notUsed0 /* TODO: gouse */
		used0    bool
	)
	notUsed1, used1 := "", ""; _ = notUsed1 /* TODO: gouse */
	_, _ = used0, used1
}
